src/data-access/impl.ts

```diff
@@ -241,7 +241,8 @@ export class ArchiverDataAccess implements DataAccess{
       this.host = window.location.host.indexOf(":") !== -1 ? window.location.host.split(":")[0] : window.location.host;
     }
 
-    if (window.location.host === "localhost:8080" || window.location.host === "127.0.0.1:8080") {
+    if (window.location.host === "localhost:8080" || window.location.host === "127.0.0.1:8080" ||
+          window.location.host === "10.20.31.48:8080") {
```

Comment:
What does this IP means? Wouldn't it be better to define a variable with an explanation of which machine it belongs?

Reply:
These IPs are for local testing. This is my computer's IP, with this IP we can test in any of the computers connected to the same network as mine.